src/main/java/io/stargate/sgv2/jsonapi/service/operation/reranking/RerankingMetrics.java

```diff
@@ -0,0 +1,56 @@
+package io.stargate.sgv2.jsonapi.service.operation.reranking;
+
+import static io.stargate.sgv2.jsonapi.util.ClassUtils.classSimpleName;
+
+import io.micrometer.core.instrument.*;
+import io.stargate.sgv2.jsonapi.api.request.RequestContext;
+import io.stargate.sgv2.jsonapi.service.reranking.operation.RerankingProvider;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class RerankingMetrics {
+  final String RERANKING_NUMBER_OF_PASSAGE = "reranking.passage.count";
+  final String RERANKING_CALL_DURATION_METRICS = "reranking.call.duration";
+  final String RERANKING_PROVIDER_METRICS_TAG = "reranking.provider";
```

Comment:
this tag name looks like a metric name, the use of "." seems unnecessary. 

can we use provider and model as tags ?